slate-note-taking/src/Components/Editor/RTEEditor.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import "react-quill/dist/quill.snow.css";
+import ReactQuill from "react-quill";
+import "./RTEEditor.css";
+import { useNoteTakingContext } from "../Context/NotetakingContext";
+function RTEEditor() {
+  const { textareaBoxValue, notesTakingFn } = useNoteTakingContext();
+
+  return (
+    <div>
+      <ReactQuill
+        theme="snow"
+        required
+        value={textareaBoxValue}
+        onChange={(event) =>
+          notesTakingFn({ type: "TEXTAREABOXVALUE", payload: event })
```

Comment:
the type of text is not readable. Add Some hyphens between the text so it will be more readable